plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -220,8 +224,18 @@ public MetadataEntry getMetadataEntry(TableSnapshot tableSnapshot, ConnectorSess
     }
 
     public List<AddFileEntry> getActiveFiles(TableSnapshot tableSnapshot, MetadataEntry metadataEntry, ProtocolEntry protocolEntry, ConnectorSession session)
+    {
+        return getActiveFiles(tableSnapshot, TupleDomain.all(), metadataEntry, protocolEntry, session);
+    }
+
+    public List<AddFileEntry> getActiveFiles(TableSnapshot tableSnapshot, TupleDomain<DeltaLakeColumnHandle> partitionConstraint, MetadataEntry metadataEntry, ProtocolEntry protocolEntry, ConnectorSession session)
```

Comment:
nit: partitionConstraint argument should IMO be after `MetadataEntry metadataEntry, ProtocolEntry protocolEntry,`